### PR TITLE
release-notes-extractor 0.0.1

### PR DESCRIPTION
**Pull request by ashrafDoubleO7**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by ahmedsalemelzeiny** [comment]
Nice idea

File: steps/release-notes-extractor/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
[the remark this answers is not in the document]
Comment (ashrafDoubleO7) [reply steps/release-notes-extractor/0.0.1/step.yml]: Yup, you're right, missed that part!

File: steps/release-notes-extractor/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      - If it's in a relative path to your project directory, type that relative file path.
+    is_required: false
+    summary: The path of your changelog file
+    title: Changelog file path
+- keep_markdown_output: "No"
+  opts:
+    description: |
+      - If set to 'Yes', the output will remain exactly as in the changelog.
+      - If set to 'No', the output will be trimmed of all '###' prefixes before change types.
+    is_required: false
```
[the remark this answers is not in the document]
Comment (ashrafDoubleO7) [reply steps/release-notes-extractor/0.0.1/step.yml]: If no value is specified I'm putting a "No" as a default.

> - keep_markdown_output: "No"

Am I missing something?

[the remark this answers is not in the document]
Comment (koral--) [reply steps/release-notes-extractor/0.0.1/step.yml]: So it seems that `is_required` should be set to true.

[the remark this answers is not in the document]
Comment (ashrafDoubleO7) [reply steps/release-notes-extractor/0.0.1/step.yml]: After the reading the input docs, I figured you're right, changed it.

**Pull request closed without merging**